deploy/kubernetes/helm-charts/altinn-designer/values.yaml

```diff
@@ -152,8 +152,56 @@ volumes:
     secret:
       secretName: altinn-appsettings-secret
 
-service:
-  ports:
-    - port: 5000
-      protocol: TCP
-      targetPort: 5000
+ingress:
+  - metadata:
+      name: altinn-designer-base-path
+      annotations:
+        # traefik.ingress.kubernetes.io/request-modifier: AddPrefix: /users
+        # traefik.ingress.kubernetes.io/rewrite-target: /users
+        # traefik.ingress.kubernetes.io/rule-type: PathPrefixStrip
+        traefik.ingress.kubernetes.io/rule-type: Path
+        traefik.frontend.priority: "1000"
+        # kubernetes.io/ingress.class: nginx
+        # kubernetes.io/tls-acme: "true"
```

Comment:
The "commented out"-code should be kept for future reference or removed?

Reply:
It was for reference, but I can remove it.

Reply:
Removed.